luajit21.py

```diff
@@ -2567,10 +2567,7 @@ def invoke (self, args, from_tty):
         # step 3: Figure out the size of misc data structures
         strhash_size = (strmask(g) + 1) * typ("GCRef").sizeof
 
-        if is_gc64():
-            g_tmpbuf_sz = g['tmpbuf']['e']['ptr64'] - g['tmpbuf']['b']['ptr64']
-        else:
-            g_tmpbuf_sz = g['tmpbuf']['e']['ptr32'] - g['tmpbuf']['b']['ptr32']
+        g_tmpbuf_sz = int(g['tmpbuf']['e']) - int(g['tmpbuf']['b'])
```

Comment:
We should determine the type of `g['tmpbuf']['e']` and then handle it accordingly based on its type. 

There must be a reason why the code was originally written this way.